Mixpanel/Mixpanel.m

```diff
@@ -254,9 +254,10 @@ + (void)assertPropertyTypesInArray:(NSArray *)arrayOfProperties depth:(NSUIntege
 
 - (NSString *)defaultDistinctId
 {    
-    NSString *distinctId = [self IFA];
+    NSString *distinctId;
 
 #if !defined(MIXPANEL_WATCHOS) && !defined(MIXPANEL_MACOS)
+    distinctId = [self IFA];
```

Comment:
Ah! Of course you are correct. Sorry I did not see this earlier. I have modified the commit to explicitly initialize distinctId for macOS and non-macOS.